src/content/it/fundamentals/performance/rendering/simplify-paint-complexity-and-reduce-paint-areas.md

```diff
@@ -0,0 +1,203 @@
+rappresenta il tempo necessario per il disegno) e per tutte queste aree puoi
+cercare di risolvere evitando il paint quando possibile.
+
+## Promuovi elementi con movimento o dissolvenza
+
+Il Paint non viene sempre eseguito per singola immagine in memoria. In effetti è
+possibile che il browser disegni più immagini o livelli di composizione se
+necessario.
+
+<img src="images/simplify-paint-complexity-and-reduce-paint-areas/layers.jpg"
+alt="A representation of compositor layers.">
+
+Il vantaggio di questo approccio è che gli elementi che vengono ridisegnati
+regolarmente o che si spostano sullo schermo con le trasformazioni possono
+essere gestiti senza influenzare altri elementi. È lo stesso dei pacchetti
+grafici come Sketch, GIMP o Photoshop, in cui i singoli livelli possono essere
+gestiti e composti l'uno sopra l'altro per creare l'immagine finale.
+
+Il modo migliore per creare un nuovo livello è utilizzare la proprietà CSS
+`will-change` . Funziona in Chrome, Opera e Firefox e con un valore `transform`,
+creerà un nuovo livello di composizione:
+
+```
+.moving-element {
+  will-change: transform;
+}
+```
+
+Per i browser che non supportano `will-change` ma beneficiano della creazione di
+livelli come Safari e Safari mobile è necessario (non corretto) utilizzare una
+trasformazione 3D per forzare un nuovo livello:
+
+```
+.moving-element {
+  transform: translateZ(0);
+}
+```
+
+Bisogna tuttavia fare attenzione a non creare troppi livelli poiché ogni livello
+richiede sia memoria che gestione. Vi sono ulteriori informazioni su questo
+argomento nella sezione [incoraggiare le proprietà di solo compositore e
+conteggio della gestione
+livelli](stick-to-compositor-only-properties-and-manage-layer-count).
+
+Se hai promosso un elemento in un nuovo livello usa DevTools per confermare che
+questo ti dia un vantaggio in termini di prestazioni. **Non promuovere elementi
+senza profiling.**
+
+## Ridurre le aree di paint
+
+Tuttavia a volte, nonostante la promozione degli elementi, il lavoro di paint è
+ancora necessario. Una grande sfida dei problemi di paint consiste nel fatto che
+i browser uniscono insieme due aree che richiedono il paint con il risultato di
+un ridisegno dell'intero schermo. Ad esempio se hai un'intestazione fissa nella
+parte superiore della pagina e qualcosa viene dipinto nella parte inferiore
+dello schermo l'intero schermo potrebbe finire per essere ridisegnato.
+
+Note: sugli schermi ad alto DPI gli elementi che sono in posizione fissa vengono
+automaticamente promossi al proprio livello di composizione. Questo non è il
+caso dei dispositivi a basso DPI perché la promozione cambia il rendering del
+testo da subpixel in scala di grigi e la promozione del livello deve essere
+eseguita manualmente.
+
+Ridurre le aree di paint è spesso il caso di orchestrare le animazioni e le
+transizioni in modo che non si sovrappongano più o di trovare modi per evitare
```

Comment:
transizioni in modo che non si sovrappongano più di tanto o trovare modi per evitare